docs/src/pages/components/data-grid/rendering/DensitySelectorHiddenGrid.js

```diff
@@ -0,0 +1,17 @@
+import * as React from 'react';
+import { DataGrid } from '@material-ui/data-grid';
+import { useDemoData } from '@material-ui/x-grid-data-generator';
+
+export default function DensitySelectorHiddenGrid() {
+  const { data } = useDemoData({
+    dataSet: 'Commodity',
+    rowLength: 4,
+    maxColumns: 6,
+  });
+
+  return (
+    <div style={{ height: 300, width: '100%' }}>
+      <DataGrid {...data} hideToolbar={false} disableDensitySelector />
```

Comment:
```suggestion
      <DataGrid {...data} disableDensitySelector />
```

Isn't this a NOOP? (More below...)

Reply:
@mbrookes It's not a noop, see #637

Reply:
Okay, thanks. Confusing.

Reply:
Can I resolve this? For now we need to keep the `hideToolbar={false}` for the example

Reply:
I think that we should document the opposite, make the toolbar opt-in and document how to render the density toolbar.

```jsx
import * as React from 'react';
import { DataGrid, GridToolbar, GridToolbarDensitySelector } from '@material-ui/data-grid';
import { useDemoData } from '@material-ui/x-grid-data-generator';

function Header() {
  return (
    <GridToolbar>
      <GridToolbarDensitySelector />
    </GridToolbar>
  );
}

export default function DensitySelectorToolbar() {
  const { data } = useDemoData({
    dataSet: 'Commodity',
    rowLength: 10,
    maxColumns: 6,
  });

  return (
    <div style={{ height: 400, width: '100%' }}>
      <DataGrid {...data} components={{ Header }} density="compact" />
    </div>
  );
}
```